docs/LOCAL_DEPLOYMENT.md

```diff
@@ -2,6 +2,8 @@
 
 # Local setup
 
+> **Note for macOS Developers**: If you are using macOS on Apple Silicon (ARM64) the DevContainer will **not** work. This is due to a limitation with the Azure Functions Core Tools. We recommend using the [Non DevContainer Setup](./NON_DEVCONTAINER_SETUP.md) instructions to run the accelerator locally.
```

Comment:
Can we link to the limitation around the `Azure Functions Core Tools`, is there any issue or any docs for it?

Reply:
Good shout - updated